client/layout/head/index.jsx

```diff
@@ -0,0 +1,52 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Helmet from 'react-helmet';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import decodeEntities from 'lib/formatting/decode-entities';
+
+const debug = debugFactory( 'calypso:layout:head' );
+
+const Head = ( { title, isTitleFormatted, description, canonicalUrl, children } ) => (
+	<div>
+		<Helmet
+			title={ isTitleFormatted ? title : buildTitle( title ) }
```

Comment:
Maybe we should go for whatever's simplest, hardcoding seems OK to me for now, since we don't need to localise that fragment.